docs/gen-ai/README.md

```diff
@@ -0,0 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: AI
+path_base_for_github_subdir:
+  from: content/en/docs/specs/semconv/ai/_index.md
+  to: gen-ai/README.md
+--->
+
+# Semantic Conventions for Generative AI systems
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for the following kind of Generative AI systems:
```

Comment:
Nit:

what do you think about adding a disclaimer saying that we're early in the process, everything can change, it's not a complete MVP, and that we want feedback?

something like 

> **Warning**
> The semantic conventions for GenAI and LLM are currently in development.
> We encourage instrumentation libraries and telemetry consumers developers to 
> use the conventions in limited non-critical workloads and share the feedback